bin/git_rails

```diff
@@ -19,7 +19,7 @@ if [ ! -z "$bundle_changed" ]; then
   bundle
 fi
 
-migrations=`git diff --name-status $old_ref $new_ref -- db/migrate`
+migrations=`git diff --name-status $old_ref $new_ref -- "db/migrate*"`
```

Comment:
It was pointed out to me offline that migrations are no longer run in the correct order if the folders are different.